certified-connectors/Tesseron ASM Ticket/apiDefinition.swagger.json

```diff
@@ -0,0 +1,819 @@
+                  "description": "Message"
+                },
+                "Success": {
+                  "type": "boolean",
+                  "description": "Success"
+                },
+                "TicketNumber": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "description": "Ticket number of the edited ticket",
+                    "title": "Ticket Number"
+                  },
+                  "description": "TicketNumber"
+                },
+                "TicketPositionNumber": {
+                  "type": "string",
+                  "description": "ID of the created ticket position",
+                  "title": "Ticket Position ID"
+                }
+              }
+            },
+            "headers": {
+              "{": {
+                "description": "{",
+                "type": "string"
+              },
+              "\"cache-control\"": {
+                "description": "\"cache-control\"",
+                "type": "string"
+              },
+              "\"content-length\"": {
+                "description": "\"content-length\"",
+                "type": "string"
+              },
+              "\"content-security-policy\"": {
+                "description": "\"content-security-policy\"",
+                "type": "string"
+              },
+              "\"content-type\"": {
+                "description": "\"content-type\"",
+                "type": "string"
+              },
+              "\"date\"": {
+                "description": "\"date\"",
+                "type": "string"
+              },
+              "\"referrer-policy\"": {
+                "description": "\"referrer-policy\"",
+                "type": "string"
+              },
+              "\"strict-transport-security\"": {
+                "description": "\"strict-transport-security\"",
+                "type": "string"
+              },
+              "\"x-frame-options\"": {
+                "description": "\"x-frame-options\"",
+                "type": "string"
+              },
+              "\"x-ms-apihub-cached-response\"": {
+                "description": "\"x-ms-apihub-cached-response\"",
+                "type": "string"
+              },
+              "\"x-ms-apihub-obo\"": {
+                "description": "\"x-ms-apihub-obo\"",
+                "type": "string"
+              },
+              "\"x-xss-protection\"": {
+                "description": "\"x-xss-protection\"",
+                "type": "string"
+              },
+              "\"_x-content-type-options\"": {
+                "description": "\"_x-content-type-options\"",
+                "type": "string"
+              },
+              "}": {
+                "description": "}",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "Create Ticket Position",
+        "description": "CreateTicketPosition",
```

Comment:
@developmentLL descriptions with CaMel case are considered non-translatable. Please make sure that descriptions and titles look like valid sentences. More information about English strings can be found in [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).